model/shared/src/main/scala/explore/model/formats.scala

```diff
@@ -128,7 +128,7 @@ trait formats:
             "Duration parsing errors".refined[NonEmpty]
           }
           .toEitherErrors,
-      ts => f"${ts.toHoursPart}:${ts.toMinutesPart}%02d"
+      ts => f"${ts.toHours.toLong}:${ts.toMinutesPart}%02d"
```

Comment:
Why this?

Reply:
Same reason as the comment above: so that it doesn't get truncated if you express a duration of more than 24 hours. `hoursPart` is bounded to `[0, 23]`.